### ISSUE TITLE
[Bug]: RovingFocus should have no default `orientation`

### ISSUE THREAD

**Issue by zernonia** [radix-vue/radix-vue#382]
@khairulhaaziq :

I found the radio has accessibility issue where the orientation should be vertical instead of horizontal, https://www.shadcn-vue.com/docs/components/radio-group.html, and then I found out default orientation is undefined https://github.com/radix-vue/radix-vue/blob/044407eb44ebdfe525accbb5fb8a6be113d838fa/packages/radix-vue/src/RadioGroup/RadioGroupRoot.vue#L74 here, which then inherit from the horizontal in roving focus. I wonder if we should add default orientation as vertical in radix-vue radiogroup, or just add it in shadcn radiogroup
